src/Nethermind/Nethermind.Synchronization/FastSync/DetailedProgress.cs

```diff
@@ -130,41 +132,35 @@ private void LoadFromSerialized(byte[] serializedData)
 
         public byte[] Serialize()
         {
-            int contentLength = GetLength();
-            RlpStream stream = new(Rlp.LengthOfSequence(contentLength));
+            long[] progress = new long[]
```

Comment:
let's do a stackalloc and span if we really need it. But would prefer to keep it in an explicit format.